### DISCUSSION TITLE
Make the new recommendations disappear

### DISCUSSION THREAD

**Discussion by Moxinilian** [community/community#54137] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be nice if one could opt out of the recommendation system. I only want to use GitHub for work, not as a social network.

**Comment by casperriboe** [comment]
There is a filter button at the top right. Click on it and uncheck "Recommendations" at the bottom of the list.

**Reply by Moxinilian** [reply]
I meant something to get rid of unrelated activity entirely, I do not know what they call this. Even when filtering recommendations I still get things about people I do not know about.
